docs/tutorials/gluon/hybrid.md

```diff
@@ -1,6 +1,9 @@
 # Hybrid - Faster training and easy deployment
 
-*Note: a newer version is available [here](http://gluon.mxnet.io/chapter07_distributed-learning/hybridize.html).*
+*Related Contents:*
```

Comment:
nit: Content instead of Contents (but don't restart CI just for that change).

Reply:
I added a link to the [new dive into deeplearning book](http://en.diveintodeeplearning.org/chapter_computational-performance/hybridize.html) so made it "contents"

Reply:
It's a semantic thing I guess. There's probably a rule here but... usually you can say Content is plural already (as it refers to a mass of stuff) and you don't usually say "Related Contents". It's more common to say "Related Content" even if there's more than one item in the list.
It is weird because you do say "Table of Contents", not "Table of Content". 
🤷‍♂️ English is weird. It's not a showstopper... it's fine whichever way you want to do it. (but do it my way). Jk